DigitalLearningSolutions.Web/Views/Shared/Components/CentreInfo/Default.cshtml

```diff
@@ -0,0 +1,7 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common

```

Comment:
I'm worried I might be the cause of the LF issues we're having, since I made this file and it, CentreInfoViewComponent and CentreInfoViewModel apparently had LF line endings. They've all got CRLF now.
Interestingly the latest file I made, centreInfo.scss has the correct line endings without me doing anything.

Reply:
Having files as only LF shouldn't cause issues as long as we aren't changing them backwards and forwards each time. Sounds very similar to some of the issue I've been having, some of the files I've created are also LF and I have no idea why.

After looking at some of them that I was having issues with yesterday, it seems like merging was partially causing some of the issues. An LF file from master being merged into a local CRLF was resulting in mixed line endings. we're just going to keep an eye on them and try and make them consistent where possible (e.g. I'm going to go through and change a load of them before merging 404 after it has been reviewed so the review isn't confusing as to what has changed)